policy/resolver.go

```diff
@@ -376,11 +376,12 @@ type resolverCache struct {
 	riskInfos        map[string]*RiskFactor
 	propsCache       explorer.PropsCache
 
-	reportingJobsByUUID map[string]*ReportingJob
-	reportingJobsByMsum map[string][]*ReportingJob // Msum == Mquery.Checksum, i.e. only reporting jobs for mqueries
-	reportingJobsActive map[string]bool
-	errors              []*policyResolutionError
-	bundleMap           *PolicyBundleMap
+	reportingJobsByUUID   map[string]*ReportingJob
+	reportingJobsByMsum   map[string][]*ReportingJob // Msum == Mquery.Checksum, i.e. only reporting jobs for mqueries
+	reportingJobsByCodeId map[string][]*ReportingJob // CodeId == check.CodeId
```

Comment:
replace `check` in the comment with either `Mquery` (to be consistent with the above) or query. ReportingJobs don't mind if it's a check or a data query

Reply:
Thanks for the feedback, changed this.